Java: Add Webflux integration docs. (#3998)
getsentry/sentry-docs: src/platforms/java/guides/spring-boot/webflux.mdx

---

> title: Webflux Integration

Reviewer: I've made edits to this new page, but I'm concerned that there's barely any content here. Will there be more added at some point? If not, can this page possibly be a small section on another page in the Spring Boot guide? 

```suggestion
title: WebFlux Integration
```

sidebar_order: 6

> description: "Learn more about using Spring Webflux with Sentry."

Reviewer: ```suggestion
description: "Learn more about using Spring WebFlux with Sentry."
```

---

> Similar to Spring MVC, Sentry has a support for Spring Webflux. When developing a reactive web application, Sentry Spring Boot Starter creates infrastructure required for capturing errors triggered during HTTP request processing.

Reviewer: ```suggestion
Similar to Spring MVC, Sentry offers support for Spring WebFlux. When developing a reactive web application, Sentry Spring Boot Starter creates the infrastructure required for capturing errors triggered during HTTP request processing.
```

<Note>

> Webflux support is available from version 5.1.0 in experimental phase. The API as well as internals may change in the future releases.

Reviewer 2: that's fine, but we have a "predefined" wording for that (its part of our docs guidelines):

> Support for Flutter's Desktop is available as a Beta. Features available in Beta are still in-progress and may have bugs. We recognize the irony.

we can either adapt the message above or keep it as it is, your call.

Author: I'll adapt

> Support for Spring Webflux is available as a Beta. Features available in Beta are still in-progress and may have bugs. We recognize the irony.

Reviewer: ```suggestion
Support for Spring WebFlux is available as a Beta. Features available in Beta are still in-progress and may have bugs. We recognize the irony.
```
